### PR TITLE
Updated docs

### PR DESCRIPTION
**Pull request by andrii-ivaniuk**
I think we should not expose `cached_sess.request()` to users and encourage them to use verb-related methods.

### CODE REVIEW

File: src/acachecontrol/acachecontrol.py [hunk 1]
```diff
@@ -14,32 +14,32 @@ def __init__(
         self.cache = cache
         self._async_client_session = aiohttp.ClientSession()
 
-    def request(self, method, url, **params):
+    def _request(self, method, url, **params):
```
Comment (MasterSergius) [inline src/acachecontrol/acachecontrol.py]: What do you think if we will have methods "request" and "_request" as it in aiohttp?

**Review by MasterSergius — APPROVED** [review]

File: src/acachecontrol/acachecontrol.py (hunk 1 above)
Comment (andrii-ivaniuk) [reply src/acachecontrol/acachecontrol.py]: What's the reason?

File: src/acachecontrol/cache.py [hunk 2]
```diff
@@ -30,9 +28,10 @@ class AsyncCache:
     Key: Tuple(http_method, url), value: aiohttp response obj
     """
 
-    def __init__(self, config={}):
+    def __init__(self, config: Dict = None):
```
Comment (andrii-ivaniuk) [inline src/acachecontrol/cache.py:31]: @MasterSergius Why do we need `config` if a user is not able to provide it?

File: src/acachecontrol/acachecontrol.py (hunk 1 above)
Comment (MasterSergius) [reply src/acachecontrol/acachecontrol.py]: To support method request :)
Of course, somebody may use _request for that purpose, maybe it's not important for now

File: src/acachecontrol/cache.py (hunk 2 above)
Comment (MasterSergius) [reply src/acachecontrol/cache.py:31]: Why not able?
Let's assume we have some cache_config:
```py
    async with AsyncCacheControl(cache=AsyncCache(config=cache_config)) as cached_sess:
        async with cached_sess.request('GET', 'http://example.com') as resp:
            resp_text = await resp.text()
            print(resp.status)
            #print(resp.response.headers)
```

**Review by MasterSergius — APPROVED** [review]

**Pull request merged**